process-dir.js

```diff
@@ -239,18 +286,34 @@ async function process (dir) {
   })
   let files = sources.map(([file]) => file)
 
+  let nodeEnvReplacementTargets = findNodeEnvConditionsReplacementCandidates(
+    sources
+  )
+
   await Promise.all(
     sources.map(async ([file, source]) => {
       if (file.endsWith('index.browser.js')) {
-        await replaceToESM(dir, file, source)
+        let [, modifiedSource] = await replaceToESM(dir, file, source)
+        if (nodeEnvReplacementTargets.includes(file)) {
+          files.push(
+            ...(await replaceNodeEnvConditions(dir, file, modifiedSource))
+          )
+        }
```

Comment:
Should we remove `'index.browser.js'` after replacement?

Reply:
Yeap, if `index.browser.js` had `process.env.NODE_ENV` we should have only `index.dev.js`/`index.prod.js`